jigglypuffRL/deeprl/ddpg/ddpg.py

```diff
@@ -209,6 +209,30 @@ def learn(self):
         self.env.close()
         if self.tensorboard_log:
             self.writer.close()
+    def evaluate(self, num_timesteps=1000):
+        s = self.env.reset()
+        ep, ep_r, ep_t = 0, 0, 0
+
+        print("\nEvaluating...")
+        for t in range(num_timesteps):
+            a = self.select_action(s)
+            s1, r, done, _ = env.step(a)
+            ep_r += r
+            ep_t += 1
+
+            if done:
+                ep += 1
+                print("Ep: {}, reward: {}, t: {}".format(ep, ep_r, ep_t))
+                if self.tensorboard_log:
+                    self.writer.add_scalar("eval_episode_reward", ep_r, ep_t)
```

Comment:
I'm not sure if we need TB logs in the evaluate method. @ajaysub110 Do we need this?

I feel we can avoid this.

Reply:
Yeah agree. Evaluate is just to observe output right? We can have just reward printing and environment rendering

Reply:
Okay, I wasn't sure if it was required so I put it anyway.